120-ng-manager/contracts/openapi/v1/openapi.yaml

```
@@ -6,38 +6,40 @@ info:
   contact:
     email: contact@harness.io
 servers:
-  - url: 'https://api.harness.io'
+  - url: 'https://app.harness.io'
     description: Harness host URL.
 tags:
   - name: Organization
     description: This contains APIs for Organizations.
   - name: Org Project
     description: This contains APIs for Projects.
-    x-displayName: Project
+    x-displayName: 'Project [Beta]'
   - name: Account Secret
```

```suggestion
  - name: Account Secrets
```
* We might want to consider using sentence case globally ("Account secrets" instead of "Account Secrets" and "Account connectors" instead of "Account Connectors". Too much uppercase is distracting. The docs team uses sentence case in titles. This is an idea we might want to explore for API docs as well. However, let's discuss before incorporating.).

---

Okay, please skip. Thanks.